src/framework/piral-base/src/strategies.test.ts

```diff
@@ -187,12 +187,12 @@ describe('Piral-Base strategies module', () => {
     const pilets = true as any;
     const invalidLoadPiletOptions: LoadPiletsOptions = {
       createApi: createMockApi,
-      fetchPilets: jest.fn(() => Promise.resolve(pilets)),
+      fetchPilets: jest.fn(() => Promise.resolve([])),
```

Comment:
Hm this replaces the formerly invalid input with valid one - making the test rather useless.

Reply:
I tried to follow the test behavior and may have misunderstood what was considered "invalid input".  I think I considered the `pilets: true` property value in the next line to be the "invalid input".

IIRC, if the `pilets` property is invalid (because it has the value `true`), a rejection is not made at that point.  It will just call `fetchPilets`.  And when `fetchPilets` resolves to returning `true`, then an Error rejection is thrown causing the test to abort and fail.

It probably depends on which "invalid input" we want to test -- the `pilet` property or the `fetchPilets` property?

Reply:
Well, the invalid input was put there on purpose (as you may have guessed). So the test itself should not be changed in that regard.